samples/Actor/DemoActor/Program.cs

```diff
@@ -31,9 +31,9 @@ public static void Main(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
                 .UseStartup<Startup>()
-            .UseActors(actorRuntime =>
-            {
-                actorRuntime.RegisterActor<DemoActor>();
-            });
+                .UseActors(actorRuntime =>
+                {
+                    actorRuntime.RegisterActor<DemoActor>();
+                });
```

Comment:
You should see some conflicts for this file, the registration of actors has moved to `Startup.cs`.